src/dbnode/encoding/tile/counter_resets_downsampler_prop_test.go

```diff
@@ -21,27 +21,42 @@
 package tile
 
 import (
+	"fmt"
 	"math"
 	"testing"
+	"time"
 
 	"github.com/leanovate/gopter"
 	"github.com/leanovate/gopter/gen"
 	"github.com/leanovate/gopter/prop"
 )
 
+func TestDownsampleCounterResetsInvariants(t *testing.T) {
+	testDownsampleCounterResetsInvariants(t, false)
+}
+
+func TestDownsampleCounterResetsInvariantsWithPrevFrameLastValue(t *testing.T) {
+	testDownsampleCounterResetsInvariants(t, true)
+}
+
 func testDownsampleCounterResetsInvariants(t *testing.T, usePrevFrameLastValue bool) {
+	seed := time.Now().UnixNano()
 	params := gopter.DefaultTestParameters()
 	params.MinSize = 1
 	if usePrevFrameLastValue {
 		params.MinSize++
 	}
 	params.MinSuccessfulTests = 10000
 	params.MaxShrinkCount = 0
+	params.Rng.Seed(seed)
 	properties := gopter.NewProperties(params)
 	generator := gen.SliceOf(gen.Float64Range(0, 100))
 
 	epsilon := 0.00001
 
+	// NB: capture seed to be able to replicate failed runs.
+	fmt.Printf("Running tests with seed %d, usePrevFrameLastValue=%t", seed, usePrevFrameLastValue)
```

Comment:
No endline at the end of this, probably better to use a logger here?

Can grab one with:
```go
logger := instrument.NewTestOptions(t).Logger()
```

Reply:
👍